datatypes-cose/src/commonMain/kotlin/at/asitplus/crypto/datatypes/cose/CoseSigned.kt

```diff
@@ -46,22 +63,22 @@ data class CoseSigned(
             if (other.payload == null) return false
             if (!payload.contentEquals(other.payload)) return false
         } else if (other.payload != null) return false
-        return signature.contentEquals(other.signature)
+        return rawSignature != other.rawSignature
     }
 
     override fun hashCode(): Int {
         var result = protectedHeader.hashCode()
         result = 31 * result + (unprotectedHeader?.hashCode() ?: 0)
         result = 31 * result + (payload?.contentHashCode() ?: 0)
-        result = 31 * result + signature.contentHashCode()
+        result = 31 * result + rawSignature.hashCode()
```

Comment:
contentHashCode, vs. Hashcode?

Reply:
basically boils down to if we want to hash the padded raw signature or the non-padded asn1 element. what do you prefer?